src/main/java/com/odysseusinc/arachne/portal/model/factory/ArachneUserFactory.java

```diff
@@ -53,4 +53,13 @@ public static ArachneUser create(IUser user) {
         );
     }
 
+    public static ArachneUser createExternalDummy(ExternalLogin login) {
+        String email = login.getEmail();
+        // TODO should it have any roles?
+        List<GrantedAuthority> authorities = Collections.emptyList();
+        // TODO something better here
+        long id = -login.getId();
```

Comment:
hmm, that looks like magic here. Can you add an explanation here?

Reply:
Method removed